packages/zeebe-element-templates-json-schema/resources/schema.json

```diff
@@ -295,6 +295,7 @@
                           "enum": [
                             "zeebe:input",
                             "zeebe:output",
+                            "zeebe:property",
```

Comment:
We also need to adjust our [custom error messages](https://github.com/camunda/element-templates-json-schema/blob/master/packages/zeebe-element-templates-json-schema/resources/error-messages.json) (allowed binding types, allowed optional), as they are not part of the raw schema (on purpose, cf. https://github.com/camunda/element-templates-json-schema/issues/56).

Reply:
Fixed via https://github.com/camunda/element-templates-json-schema/pull/69/commits/444ee90326e77142d327e8b01ccc2a7c9d8f9f51 I believe!